.github/workflows/image-version-update.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025 Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+
+# Reusable workflow to perform image version update on Golang based projects
+name: Image Version Update
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to release (major, minor, patch) Ex: minor"
+        required: true
+  repository_dispatch:
+    types: [image-update-workflow]
+
+jobs:
+  # image version update
+  image-version-update:
+    uses: dell/common-github-actions/.github/workflows/image-version-workflow.yaml@main
+    with:
+      version: "${{ github.event.inputs.version || 'minor' }}"
```

Comment:
So, one can provide a full version or just the minor one?

Reply:
I think the inputs would only be major, minor, or patch. If one isn't provided, the default is minor version